client/src/utils/index.js

```diff
@@ -0,0 +1,18 @@
+const fetchInfo = async () => {
+    const response = await fetch("http://localhost:5000/get");
+    const data = await response.json();
+    return data;
+};
+
+const fetchDataLocation = async (inputLocation) => {
+    const response = await fetch("http://localhost:5000/get/location", {
```

Comment:
This url isn't right, it binds to this: https://github.com/team2-software-developers/Pathfinder/blob/b391d449f53e338e08ecf9fd33c656d26ad5c09b/src/server.js#L33 which doesn't have '/get' in the url anywhere

Reply:
ohh sorry , that was my mistake ,I fix it now.